ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -28,7 +28,9 @@ if [[ x"${architecture}" == x"arm64" ]]; then
 fi
 BOOTSTRAP_NODE_TYPE=${arch_instance_type}.large
 
-workers=3
+masters=${MASTER_REPLICAS}
+workers=${WORKER_REPLICAS}
```

Comment:
I'm not excited about having both `WORKER_REPLICAS` and `SIZE_VARIANT` as knobs to control the same `workers` setting.  If we want the ability to launch single-node clusters via this step, can we add more flavors to `SIZE_VARIANT`?  Maybe `SIZE_VARIANT=single-node`?  Then you could also make any instance-type configuration necessary to handle the consolidated workload.

The single-node folks presumably also have some precedent for how they like to handle CI, so probably worth checking in with them.

Reply:
Ah, and you can also adjust `mastersSchedulable` based on the same `SIZE_VARIANT` knob.

Reply:
@omertuc `single-node-conf-aws` will set `NETWORK_TYPE` to `OVNKubernetes` by default, does that mean SNO prefer OVN networking?

Reply:
@jianlinliu I happen to verify the bug this noon - https://bugzilla.redhat.com/show_bug.cgi?id=2090487, OVNKubernetes is mandatory for SNO and the default value even we didn't set networkType in the install-config.

Reply:
ah, good to know, Thanks.